pooch/__init__.py

```diff
@@ -2,7 +2,7 @@
 # Import functions/classes to make the API
 from . import version
 from .core import Pooch, create
-from .utils import os_cache, file_hash, make_registry, check_version
+from .utils import os_cache, file_hash, make_registry, check_version, get_logger
```

Comment:
I don't think we need to expose `get_logger` to users since it's only used internally. In that case, it doesn't need to be here.

Reply:
Ok, I added this for users who want to change the logging level -- something like `pooch.get_logger().setLevel("CRITICAL")`, since you can just pass a string for the level. I can remove it from the public API but I think it might be good to have another way to set the logging level then.

Reply:
Ah right, that makes perfect sense. We can leave it then.